src/renderer/extensions/vueNodes/widgets/components/audio/AudioPreviewPlayer.vue

```
@@ -137,6 +150,10 @@ import { computed, ref, useTemplateRef } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { whenever } from '@vueuse/core'
 
+import { useToast } from 'primevue'
```

ESLint

[error] Unable to resolve path to module 'primevue'.

(import-x/no-unresolved)